src/cmd/tools/dtest/util/watcher.go

```diff
@@ -142,7 +143,7 @@ func (nw *nodesWatcher) WaitUntilAll(p NodePredicate, timeout time.Duration) boo
 				return
 			case <-reporter.C:
 				numPending, pendingString := nw.pendingStatus()
-				nw.logger.Infof("%d instances remaining: [%v]", numPending, pendingString)
+				nw.logger.Info("instances remaining", zap.Int("num", numPending), zap.String("instances", pendingString))
```

Comment:
not too much context but should be `numPending`?

Reply:
sure

Reply:
done